lms/static/scripts/frontend_apps/utils/grader-service.js

```diff
@@ -1,13 +1,31 @@
 import { apiCall } from './api';
 
 /**
- * Submits the grade to the LTI endpoint, waits for it to return, then updates the state with the
- * new grade
+ * @typedef {Object} Student
+ * @prop {string} LISResultSourcedId - The unique student id
+ * @prop {string} LISOutcomeServiceUrl - The student's service url
+ */
+
+/**
+ * The fetched grade result
+ *
+ * @typedef {Promise<Object>} FetchGradeResult
+ * @prop {number} currentScore - The fetched grade
+ */
+
+/**
+ * The submitted grade result
+ *
+ * @typedef {Promise<Object>} SubmitGradeResult
```

Comment:
An empty object, then? Or am I reading this wrong (quite likely)? Looking down at the code, it looks like `submitGrade` returns the return value of `apiCall`...I feel like I might be missing something here.

Reply:
oh I think I'm getting into the weeds here, I mean it returns a Promise because its wrapped in async, but perhaps thats a bit too much under the hood and a distraction from what its really doing. In any case, I think either way is okay, so I'll get rid of the Promise gunk.